wiki/static/wiki/css/modules/_site-layout.sass

```diff
@@ -1,19 +1,15 @@
-.site-layout-sidebar
-	width: 300px
-	position: absolute
-	left: 0
-	top: 0
-	bottom: 0
-	background-color: #F8FAFC
-	border-right: 1px solid darken(#F8FAFC, 2%)
+.site-layout
+	display: flex
+	height: 100vh
```

Comment:
This might have to become min-height, but right now I envision the content scrolling on overflow instead. TBD.